docs/_data/importer_dependencies.yml

```diff
@@ -0,0 +1,109 @@
+---
+behance:
+- behance
+- safe_yaml
+blogger:
+- rexml/document
+- rexml/parsers/streamparser
+- rexml/streamlistener
+- safe_yaml
+dotclear:
+- active_support
+- active_support/core_ext/string/inflections
+- pp
+- safe_yaml
+drupal6:
+- mysql2
+- pg
+- safe_yaml
+- sequel
+drupal7:
+- mysql2
+- pg
+- safe_yaml
+- sequel
+drupal8:
+- mysql2
+- pg
+- safe_yaml
+- sequel
+easyblog:
+- mysql2
+- safe_yaml
+- sequel
+enki:
+- pg
+- sequel
+ghost:
+- safe_yaml
+- sequel
+- sqlite3
+googlereader:
```

Comment:
Since the filename changed for this importer in the docs, the "next" and "previous" links should be updated. Search the site source for "google_reader" and they should show the places it needs to be changed (remove the underscore).

Reply:
The filename changed. But the permalink did not.
So no links broken.